course/1 Fundamentals/1.1 Fundamentals of Computer Vision/defination.mdx

```diff
@@ -1,42 +1,88 @@
-Pursuing the visual world is a bit of a puzzle. Let's revisit kicking a ball
-we discussed in the previous chapter, which involves spotting it, working out the
-best way to kick it and other miniature details to complete the task. But guess what?
-Our brains can do it fast and effortlessly because of years of evolving and improving it.
-That's the essence of computer vision, enabling computers to see and understand the world as
-we do and make smart choices.
-
-Human Vision and Computer Vision models have a lot in common; in fact, much of the research in computer vision
-is conducted by studying human vision. However, they also have key differences. One of the most significant
-distinctions is that human vision systems can learn and adapt in a way that computer vision systems cannot.
-While current computer vision systems are remarkable in mimicking human vision, they lack adaptive and learning
-capabilities that make human vision so versatile. Bridging this gap is a fascinating challenge for researchers
-and an area of active exploration within the field.
-
-In the upcoming chapters, you'll delve deeper into some of the renowned computer vision architectures,
-some of which are influenced by studying human vision. But for now, let's understand what computer vision is.
-
```

Comment:
I intended to write the second paragraph for "Human vs computer vision" without actually mentioning it since i felt too many side heading confuses readers so i gave a quick intro and started navigating into Human vs Computer Vision but i do agree "much of the research in CV is conducted by studying the human vision" dont make sense or even false considering the facts and thanks for bringing this up.
I'm open to whatever suggestion you gave like i can rewrite the whole paragraph there or rewrite and introduce a new side heading " Human vs Computer vision"